docs-source/content/userguide/cicd/choose-an-approach.md

```diff
@@ -10,44 +10,69 @@ Let's review what we have discussed and talk about when we might want to use
 various approaches.  We can start by asking ourselves questions like these:
 
 
-- *Can you make the desired change with a configuration override?*  
-  The WebLogic Kubernetes Operator allows you to inject a number of [configuration
+- *Can you make the desired change with a configuration override or Model in Image config map?*  
+
+  When your domain source type is Domain in PV or Domain in Image,
+  the WebLogic Kubernetes Operator allows you to inject a number of [configuration
   overrides]({{< relref "/userguide/managing-domains/configoverrides/_index.md" >}})
-  into your pods before starting any servers in the domain.  This allows you to use 
-  the same image for multiple
-  different configurations.  A good example would be changing the settings for a data
-  source, for example. You may wish to have a larger connection pool in your production
+  into your pods before starting any servers in the domain.  
+
+  When your domain
+  source type is Model in Image you can 
+  [inject model updates]({{< relref "/userguide/managing-domains/model-in-image/runtime-updates.md" >}})
+  before starting any servers in the domain for any type of update, or, even while your domain
+  is running for most types of updates. Model in Image runtime model updates are propagated
+  by [restarting]({{< relref "/userguide/managing-domains/domain-lifecycle/restarting.md" >}})
+  (rolling) the running WebLogic servers.
+
+  A good example of a change would be changing the settings for a data
+  source. For example, you may wish to have a larger connection pool in your production
   environment than you do in your development/test environments.  You probably also
   want to have different credentials.  You may want to change the service name, and
-  so on.  All of these kinds of updates can be made with configuration overrides.
+  so on. All of these kinds of updates can be made with configuration overrides
+  for Domain in PV and Domain in Image, and with model updates for Model in Image.
   These are placed in a Kubernetes config map, that is, they are outside of the image, so
   they do not require rebuilding the Docker image.  If all of your changes fit into
   this category, it is probably much better to just use configuration overrides
   instead of building a new image.  
+
 - *Are you only changing the WebLogic configuration, for example, deploying or updating an
   application, changing a resource configuration in a way that is not supported by
-  configuration overrides, and such?*  
+  configuration overrides, Model in Image model updates, and such?*  
```

Comment:
I am not following the question and the answers provided.  " Model in Image model updates, and such?" and then the answer only talks about "domain-in-image"

Reply:
General comment -  would it be easier to read if there is a table of 3 columns for DII, DOPV, MII and each list out the scenario or use case best fit?

Reply:
@jshum2479 

I'm inclined to leave it as is. That was my original inclination too, but it turns out that this is the style of the entire ci/cd doc set, as authored by Mark. My assumption is that the general idea is to helper the reader gain an understanding of a somewhat difficult area using gentle yet persuasive conversational English. 

Please resolve if you agree, or add further comments.  



Please resolve or comment further.

Reply:
ok, we can leave it as is.